bokeh/models/glyphs.py

```diff
@@ -462,12 +493,12 @@ class ImageURL(Glyph):
     """)
 
     angle = AngleSpec(default=0, help="""
-    The angles to rotate the images, in radians as measured from the
-    horizontal.
+    The angles to rotate the images, as measured from the horizontal.
```

Comment:
because now the units are configurable and so that doc was incorrect/misleading as stated.